io/plugins/eu.esdihumboldt.hale.io.gml/plugin.xml

```diff
@@ -480,6 +480,20 @@
                   sample="false">
             </valueDescriptor>
          </providerParameter>
+         <providerParameter
+               description="Automatically add codespace to gml:identifier during GML export."
+               label="Add codespace to gml:identifier"
+               name="xml.add.codespace"
+               optional="true">
+            <parameterBinding
+                  class="java.lang.Boolean">
+            </parameterBinding>
+            <valueDescriptor
+                  default="true"
```

Comment:
Isn't the value you set automatically INSPIRE specific? If it is enabled by default from now that also impacts non-INSPIRE datasets.